xapian-applications/omega/templates/query

```diff
@@ -37,87 +37,96 @@ $if{$opt{topterms},$include{inc/toptermsjs}}
 input[type=image] {
   border: 0px;
   }
+
+$include{styles.css}
 </style>
 <script type="text/javascript"><!--
 function f2(n) {return (n>9?"":"0")+n;}
 function write_date(time_t) {
 var d=new Date();d.setTime(time_t*1000);
 var D=d.getFullYear()+"-"+f2(d.getMonth()+1)+"-"+f2(d.getDate());
 var T=f2(d.getHours())+":"+f2(d.getMinutes())+":"+f2(d.getSeconds());
-document.write("<span title=\""+D+" "+T+"\">Modified:<br><b>"+D+"</b></span>");
+document.write("<span>Modified:<br/><time datetime=\""+D+" "+T+"\">"+D+"</time></span>");
 }
 // -->
 </script>
 </head>
-<body bgcolor="#ffffff">
-<form name=P method=get
+<body>
+<form id="query-form" name=P method=get
 action="$html{$or{$env{SCRIPT_NAME},omega}}" target="_top">
-<center>
-<input id="omega-autofocus" type=search name=P value="$html{$query}" size=65 autofocus>
+${Search Control - shows search button, options and textbox}
+<fieldset class="query-search">
+<input class="query-search" id="omega-autofocus" type=search name=P value="$html{$query}" placeholder="Search..." size=65 autofocus>
 <script>
 if (!("autofocus" in document.createElement("input")))
  document.getElementById("omega-autofocus").focus();
 </script>
-<input type=submit value="Search">
-<hr>
-$if{,$include{inc/anyalldropbox},$include{inc/anyallradio}}
+<input class="query-search" type=submit value="Search">
+</fieldset>
+${Todo: Replace radios with search buttons}
+$if{,$include{inc/anyalldropbox},
+<fieldset class="query-options">$include{inc/anyallradio}</fieldset>}
```

Comment:
Similar, this could be just `class=options`; below `class=suggested-terms` and so on.